menu/drivers/ozone/ozone_sidebar.c

```diff
@@ -686,6 +686,7 @@ void ozone_context_reset_horizontal_list(ozone_handle_t *ozone)
             and the subsequent space */
          chr          = title_noext;
          hyphen_found = false;
+         settings_t *settings = config_get_ptr();
 
          while (true)
```

Comment:
Hey, this is a cool feature!  :)

One tiny thing though - if `ozone_truncate_playlist_name` is false, there's no need to check for hyphens at all. You could omit the `while()` loop here, and save a few clock cycles. (Just a suggestion!)